### PR TITLE
DON-599: Update links to new WP pages

### PR DESCRIPTION
**Pull request by bdsl**

### CODE REVIEW

**Comment by bdsl** [comment]
See https://thebiggive.atlassian.net/browse/DON-599?focusedCommentId=28224 before merging.

File: src/app/buy-credits/buy-credits.component.html [hunk 1]
```diff
@@ -99,7 +99,7 @@ <h3 class="b-rh-1 b-bold">Purchase Donation Credits</h3>
                     <p><strong>Note:</strong>This Gift Aid section is only for claiming Gift Aid on the tip to Big Give. If you wish to claim Gift Aid on the credit amount used for making donations, you may do so when donating
                     to a charity's campaign.</p>
 
-                    <p><a href="">Find out more about Gift Aid </a></p>
+                    <p><a href="https://blog.thebiggive.org.uk/terms-and-conditions">Find out more about Gift Aid </a></p>
```
Comment (bdsl) [inline src/app/buy-credits/buy-credits.component.html:102]:
```suggestion
                    <p><a href="https://blog.thebiggive.org.uk/terms-and-conditions" target="_blank">Find out more about Gift Aid </a></p>
```

File: src/app/buy-credits/buy-credits.component.html [hunk 2]
```diff
@@ -166,7 +166,7 @@ <h3 class="b-rh-1 b-bold">Purchase Donation Credits</h3>
                     <ng-template matStepLabel>Submission</ng-template>
 
                     <p>By clicking on the 'Purchase Credits' button below, you legally agree to transfer the credit funds via bank transfer to the bank account and sort code
-                        displayed on the next screen. See our <a href="https://www.thebiggive.org.uk/s/terms-and-conditions">Terms and Conditions</a> for more information.
+                        displayed on the next screen. See our <a href="https://blog.thebiggive.org.uk/terms-and-conditions">Terms and Conditions</a> for more information.
```
Comment (bdsl) [inline src/app/buy-credits/buy-credits.component.html:169]:
```suggestion
                        displayed on the next screen. See our <a href="https://blog.thebiggive.org.uk/terms-and-conditions" target="_blank">Terms and Conditions</a> for more information.
```

**Comment by bdsl** [comment]
Closing for now because the link targets are not yet live and we don't want to create broken links. Can re-open later.

**Comment by bdsl** [comment]
Reopening for visibility, not ready to go just yet though.

File: src/app/app.component.html [hunk 3]
```diff
@@ -9,32 +9,32 @@
...
-        <li><a href="https://www.thebiggive.org.uk/s/green-match-fund">Green Match Fund</a></li>
-        <li><a href="https://www.thebiggive.org.uk/s/champions-for-children">Champions for Children</a></li>
-        <li><a href="https://www.thebiggive.org.uk/s/women-and-girls-match-fund">Women and Girls Match Fund</a></li>
-        <li><a href="https://www.thebiggive.org.uk/s/run-your-own-campaign">Run your own campaign</a></li>
+        <li><a href="https://blog.thebiggive.org.uk/match-funding-explained">Match Funding Explained</a></li>
+        <li><a href="https://blog.thebiggive.org.uk/christmas-challenge">Christmas Challenge</a></li>
+        <li><a href="https://blog.thebiggive.org.uk/green-match-fund">Green Match Fund</a></li>
+        <li><a href="https://blog.thebiggive.org.uk/champions-for-children">Champions for Children</a></li>
+        <li><a href="https://blog.thebiggive.org.uk/women-girls-match-fund">Women and Girls Match Fund</a></li>
+        <li><a href="https://blog.thebiggive.org.uk/run-your-own">Run your own campaign</a></li>
```
Comment (bdsl) [inline src/app/app.component.html:22]:
```suggestion
        <li><a href="https://blog.thebiggive.org.uk/run-your-own-campaign/">Run your own campaign</a></li>
```

File: src/app/app.component.html [hunk 4]
```diff
@@ -49,27 +49,26 @@
...
-    <li><a href="https://www.thebiggive.org.uk/s/christmas-challenge">Christmas Challenge</a></li>
-    <li><a href="https://www.thebiggive.org.uk/s/green-match-fund">Green Match Fund</a></li>
-    <li><a href="https://www.thebiggive.org.uk/s/champions-for-children">Champions for Children</a></li>
-    <li><a href="https://www.thebiggive.org.uk/s/women-and-girls-match-fund">Women and Girls Match Fund</a></li>
-    <li><a href="https://www.thebiggive.org.uk/s/run-your-own-campaign">Run your own campaign</a></li>
+    <li><a href="https://blog.thebiggive.org.uk/christmas-challenge">Christmas Challenge</a></li>
+    <li><a href="https://blog.thebiggive.org.uk/green-match-fund">Green Match Fund</a></li>
+    <li><a href="https://blog.thebiggive.org.uk/champions-for-children">Champions for Children</a></li>
+    <li><a href="https://blog.thebiggive.org.uk/women-girls-match-fund">Women and Girls Match Fund</a></li>
+    <li><a href="https://blog.thebiggive.org.uk/run-your-own">Run your own campaign</a></li>
```
Comment (bdsl) [inline src/app/app.component.html:56]:
```suggestion
    <li><a href="https://blog.thebiggive.org.uk/run-your-own-campaign/">Run your own campaign</a></li>
```

**Review by alihejazi — APPROVED** [review]

**Pull request merged**